core/src/View/MDSpan/Kokkos_MDSpan_Header.hpp

```diff
@@ -25,10 +25,13 @@ static_assert(false,
 // Look for the right mdspan
 #if __has_include(<mdspan>)
```

Comment:
Two things: 
- also check for the feature test macro so we don't pull in partial implementation like in clang right now.
- don't we need to do a `using std::layout_right/left/stride` and `using std::mdspan` too?

Reply:
done, also added in `std::default_accessor`